website/common/locales/en/generic.json

```diff
@@ -296,5 +296,6 @@
   "selected": "Selected",
   "howManyToBuy": "How many would you like to buy?",
   "habiticaHasUpdated": "There is a new Habitica update. Refresh to get the latest version!",
-  "contactForm": "Contact the Moderation Team"
+  "contactForm": "Contact the Moderation Team",
+  "testServerNoRegistration": "Test servers don't allow registration"
```

Comment:
I don't expect this message to be something the general population will see (if they do, there's already been at least a minor goof somewhere). I don't think this needs to be a translated string, then, though maybe @paglias can weigh in.